Product.API/Program.cs

```diff
@@ -11,15 +14,24 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 builder.Services.AddDbContext<ApplicationDbContext>(options => options.UseSqlServer(
-    "name=ConnectionStrings:DefaultConnection"));
+    "name=ConnectionStrings:DefaultConnection"), ServiceLifetime.Transient);
+builder.Services.AddTransient<IItemService, ItemService>();
```

Comment:
create a different file where in keep all the dependencies and call that file here